docs/pages/guides/CssOverrides.example.tsx

```diff
@@ -1,37 +1,38 @@
 import React, { useState } from 'react';
-import lightBlue from '@material-ui/core/colors/lightBlue';
+import isWeekend from 'date-fns/isWeekend';
 import TextField from '@material-ui/core/TextField';
+import lightBlue from '@material-ui/core/colors/lightBlue';
 import { createMuiTheme, ThemeProvider } from '@material-ui/core';
 import { DatePicker, DatePickerProps } from '@material-ui/pickers';
 
 const materialTheme = createMuiTheme({
   overrides: {
-    MuiPickersToolbar: {
-      toolbar: {
+    MuiPickerToolbar: {
```

Comment:
Could a version without an `s` feels inconsistent with the name of the package? `@material-ui/pickers`

Reply:
I don't understand, it was original intent of the naming, isn't it?  (#1778) but we decided to move out?

Reply:
I don't understand either. What do you think of the last comment on #1778? If you agree with it, the intended name is 

```suggestion
    MuiPickersToolbar: {
```

Reply:
I still think that correct prefix is with a `s` to match the package name `MuiPickersToolbar`.